blocks/library/block/edit-panel/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function ReusableBlockEditPanel( props ) {
+	const { isEditing, name, isSaving, onEdit, onDetach, onChangeName, onSave, onCancel } = props;
+
+	return (
+		<div className="reusable-block-edit-panel">
+			{ ! isEditing && ! isSaving && [
+				<span key="info" className="reusable-block-edit-panel__info">
+					<b>{ name }</b>
+				</span>,
+				<Button
+					key="edit"
+					isLarge
+					className="reusable-block-edit-panel__button"
+					onClick={ onEdit }>
+					{ __( 'Edit' ) }
+				</Button>,
+				<Button
+					key="detach"
+					isLarge
+					className="reusable-block-edit-panel__button"
+					onClick={ onDetach }>
+					{ __( 'Detach' ) }
+				</Button>,
+			] }
+			{ ( isEditing || isSaving ) && [
+				<input
+					key="name"
+					type="text"
+					disabled={ isSaving }
+					className="reusable-block-edit-panel__name"
+					value={ name }
+					onChange={ ( event ) => onChangeName( event.target.value ) } />,
+				<Button
+					key="save"
+					isPrimary
+					isLarge
+					isIndicatingProgress={ isSaving }
```

Comment:
This prop does not exist on the `Button` component, and since Button passes through unrecognized props to the underlying DOM node, it results in a warning while editing:

>Warning: React does not recognize the `isIndicatingProgress` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isindicatingprogress` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Should this be the `isBusy` prop added in #3682?

Reply:
Good catch. It definitely should be `isBusy`. Fixed in a820d1cc43dddd97d44e6e756707dfc6ab56c80b.